File: CPP/Assignment4/Program4.cpp
/////////////////////////////////////////////////////////////////////////
//
// File name :   Program4.cpp
// Description : Write recursive program which display below pattern.
// Author :      Pallavi Omprakash Malewar
// Date :        27/07/2026
//
/////////////////////////////////////////////////////////////////////////

/*

    Output :    A   B   C   D   E   F

*/


#include<iostream>
using namespace std;

void Display()
{
    int i = 0;
    char ch = 'A';

    for(i = 0 ; i < 6 ; i++)
    {
        cout << ch << "\t";
        ch++;
    }
    cout<<endl;
}

int main()
{
    Display();
    return 0;
}

File: JAVA/Assignment 10/Program4.java
/////////////////////////////////////////////////////////////////////////////////////////
//                                                                                     //
// Problem Statement : Write a Java program which accept string & one characeter from  //
//                     user & remove that character from string.                       //
// Author : Pallavi Omprakash Malewar                                                  //
// Date : 4 Aug 2026                                                                   //
//                                                                                     //
/////////////////////////////////////////////////////////////////////////////////////////


/*  

    Input : India is Demo India Hello Demo
            i

    Output: IndasDemoInda   

*/


import java.util.HashMap;
import java.util.Scanner;

class MyString
{
    public String RemoveChar(String s1, char ch)
    {        
        int n = s1.length();
        String result = "";

        for(int i = 0 ; i < n ; i++)
        {
            if(s1.charAt(i) != ch)
            {
                result = result + s1.charAt(i);
            }
        }
        return result;
    }
}

class Program4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter String : ");
        String s1 = sobj.nextLine();

        System.out.print("Enter character to remove : ");
        char ch = sobj.next().charAt(0);

        MyString mobj = new MyString();

        String result = mobj.RemoveChar(s1, ch);

        System.out.println(result);
    }
}
